contracts/ConsensusUtils.sol

```diff
@@ -16,6 +16,7 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
   uint256 public constant MIN_STAKE = 3e24; // 3,000,000
   uint256 public constant CYCLE_DURATION_BLOCKS = 17280; // 24 hours [24*60*60/5]
   uint256 public constant SNAPSHOTS_PER_CYCLE = 10; // snapshot each 144 minutes [17280/10/60*5]
+  uint256 public constant DEFAULT_VALIDATOR_FEE = 10;
```

Comment:
let make it a divisible number so delegators could charge 5.5% if they want to

Reply:
what do you mean by divisible? it's just the default percentage